packages/route159/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenvNoCC,
+  unzip,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "route159";
+  version = "${finalAttrs.majorVersion}.${finalAttrs.minorVersion}";
+  majorVersion = "1";
+  minorVersion = "10";
+
+  src = ./route159_110.zip;
```

Comment:
Does using `fetchzip` might help getting rid of `nativeBuildInputs` and `unpackPhase` ?

Reply:
Not sure you can use `fetchzip` with a locally vendored file.